### PR TITLE
add csr approver strategic with systemd

### PR DESCRIPTION
**Pull request by iranzo**
Fixes # (issue)

Approves CSR once the detached cluster has been activated at destination

### CODE REVIEW

**Comment by alknopfler** [comment]
@jparrill up to you

File: deploy-spoke/csr_autoapprover.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    echo "Waiting for API..."
+    sleep 10
+done
+
+# Begin looking for and signing CSRs to activate nodes
+CLUSTER_DATA_FOLDER="/var/home/core/cluster_access_data"
+export KUBECONFIG=$(ls $CLUSTER_DATA_FOLDER/*/kubeconfig-* | head -1)
+
+if [ ! -f "${KUBECONFIG}" ]; then
+    echo "ERROR: Could not find kubeconfig file for cluster"
```
Comment (jparrill) [inline deploy-spoke/csr_autoapprover.sh:16]: Maybe it's better to:

```
echo "WARN: Could not find Kubeconfig file for cluster"
exit 0
```

Reason: 

- On day 2, the final client will need to reboot the cluster at some point, and He will see that error (maybe in a real error situation not related with this topic.

WDYT @flaper87

**Review by jparrill — COMMENTED** [review]
LGTM

File: deploy-spoke/csr_autoapprover.sh (hunk 1 above)
Comment (alknopfler) [reply deploy-spoke/csr_autoapprover.sh:16]: done, disable unit and remove kubeconfig at the end of the script

**Pull request merged**